src/SharedResources/components/Label.ts

```diff
@@ -0,0 +1,24 @@
+import { SFC, createElement } from "react";
+import * as classNames from "classnames";
+
+export interface LabelProps {
+    className?: string;
+    label: string;
+    weight: number;
+    style?: object;
+    orientation?: "horizontal" | "vertical";
+}
+
+export const Label: SFC<LabelProps> = ({ children, className, label, style, weight, orientation }) => {
+    weight = (weight > 11 || weight < 1) ? 3 : weight;
+    const labelWeight = orientation === "horizontal" ? `col-sm-${weight}` : "";
```

Comment:
if the orientation is vertical, some styles needs to be applied.
for the label, use col-sm-1 and for the children div, use col-sm-12 and then the problem with the margins will be solved.

Reply:
Nice fix for the margins.

Reply:
col-sm-12 breaks appearance in pop ups
![image](https://user-images.githubusercontent.com/19818344/49133849-9c1db480-f2f2-11e8-8626-e497686410bf.png)

Reply:
I was trying with col-sm-1 in both elements (label and div with input inside) and that worked fine, try to apply **`col-sm-1`** instead of 12 :)

Reply:
That's what I have currently but it will push it to the left.

Reply:
Indeed just `col-1` solve the issue because bootstrap style just look for elements inside `form-group` that contains `col-` to apply margins. Thanks for your solution!! 👍

Reply:
@McDoyen lets skip this for now, because if we set the form layout to vertical, it will be wrong now and the old version will be Ok.

Reply:
Okay. I will leave it empty then.